app/react/components/Tip/Tooltip/Tooltip.tsx

```diff
@@ -31,6 +31,7 @@ export function Tooltip({ message, position = 'bottom', className }: Props) {
         effect="solid"
         className={clsx(styles.tooltip, className)}
         arrowColor="transparent"
+        delayHide={200}
```

Comment:
Is this change in EE too?

Reply:
removed it as I will do it as part of the other ticket.